plugin_dev/src/com/google/idea/blaze/plugin/run/BlazeIntellijPluginConfigurationType.java

```diff
@@ -145,7 +145,7 @@ private static String defaultVmOptions() {
       if (vmoptions == null) {
         return null;
       }
-      vmoptions = vmoptions.replaceAll("\\s+", " ").trim();
+      vmoptions = vmoptions.replaceAll("#.+", "").replaceAll("\\s+", " ").trim();
```

Comment:
Does this handle empty comment lines? Is that even allowed?

E.g.,
```
-Dfoo
#
-Dbar
```

Reply:
It didn't and I think you are right, it should. Pushed a new diff to add that.